riscv/triggers.h

```diff
@@ -89,6 +89,12 @@ class trigger_t {
   action_t legalize_action(reg_t val) const noexcept;
   reg_t tdata2;
 
+  bool vs;
+  bool vu;
+  bool m;
+  bool s;
+  bool u;
```

Comment:
These should be explicitly initialized as `= false;` like the `mcontrol_t` ones were